sentry-ruby/lib/sentry/hub.rb

```diff
@@ -104,7 +106,7 @@ def capture_exception(exception, **options, &block)
 
       capture_event(event, **options, &block).tap do
         # mark the exception as captured so we can use this information to avoid duplicated capturing
-        exception.instance_variable_set(:@__sentry_captured, true)
+        exception.instance_variable_set(Sentry::CAPTURED_SIGNATURE, true)
```

Comment:
I feel like the logic of `Sentry.exception_captured?` may be used by users or integrations in the future for checking purpose so I extracted it. But marking exception as captured should only be done here so I didn't add a method for it.